definitions/infrastructure_as_a_service.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Infrastructure as a Service (IaaS)
+status: Completed
+category: Concept
+---
+
+## Infrastructure as a Service
+
+### What it is
+
+Infrastructure as a service i.e. IaaS is a [cloud computing](https://github.com/cncf/glossary/blob/main/definitions/cloud_computing.md) service model that offers [virtualized](https://github.com/cncf/glossary/blob/main/definitions/virtualization.md) compute, storage and network resources on-demand over the internet to consumers on a pay-as-you-go cost model.
```

Comment:
I'd amend this to virtual or physical as many IaaS providers also offer physical servers.

Reply:
done